.travis.yml

```diff
@@ -2,7 +2,7 @@ language: go
 
 go:
   - 1.6
-  - tip
+  - 1.7
```

Comment:
you could always do
```
go:
  - 1.6
  - 1.7
  - tip

matrix:
  fast_finish: true
  allow_failures:
  - go: tip
```
and just toggle off allowing failures on tip mid-release

keeps travis green, lets you see tip status easily (see https://github.com/openshift/origin/pull/10472 / https://travis-ci.org/openshift/origin/builds/152847351)